src/popup/components/character-countdown.tsx

```diff
@@ -2,34 +2,49 @@ import * as React from 'react'
 
 export const CharacterCountdown = ({ characterLimit }: { characterLimit: CharacterLimit }): JSX.Element => {
   const { remaining, percentageCompleted } = characterLimit
-
   // We draw a progress circle only if the user didn't cross the limit of characters
   const drawCircle = remaining >= 0
-
   // As the user gets closer to the limit of characters, we will draw a warning circle
   const drawWarningCircle = remaining <= 20
-
   // We draw the countdown only when the user gets closer to the limit.
   // After the user crossed the limit, we draw the number of excess characters
   const drawCountdown = remaining <= 20
   const drawWarningCountDown = remaining >= 0
-
+  const strokeDasharray = 56.5487 // stole from Twitter 🤷‍♀️
+  const strokeDashoffset = strokeDasharray - strokeDasharray * (percentageCompleted / 100)
+  const radius = 9
+  const strokeWidth = 2
+  const diameter = 2 * radius + strokeWidth
   return (
-    <svg viewBox="0 0 36 36" className="circular-chart">
-      {drawCircle && (
-        <path
-          className={drawWarningCircle ? 'circle__warning' : 'circle'}
-          strokeDasharray={`${percentageCompleted}, 100`}
-          d="M18 2.0845
-        a 15.9155 15.9155 0 0 1 0 31.831
-        a 15.9155 15.9155 0 0 1 0 -31.831"
-        />
-      )}
-      {drawCountdown && (
-        <text x="18" y="22.35" className={drawWarningCountDown ? 'inner-text__warning' : 'inner-text'}>
-          {Math.abs(remaining)}
-        </text>
-      )}
-    </svg>
+    <div
+      className="character-countdown"
+      role="progressbar"
+      aria-valuemax={100}
+      aria-valuemin={0}
+      aria-valuenow={Math.round(percentageCompleted) >= 100 ? 100 : Math.round(percentageCompleted)}
```

Comment:
Looks good! We can trim this up even further by writing

```typescript
Math.min(Math.round(percentageCompleted), 100)
```

Reply:
done